frontend/src/components/WebSocket-Context.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  createContext,
+  Dispatch,
+  useContext,
+  useMemo,
+  useReducer,
+} from "react";
+
+const getThreadIdByName = (threadName: string) => {
+  return JSON.parse(localStorage.getItem(threadName)!).id;
+};
+
+const WebSocketContext = createContext<
+  | {
+      webSocketState: WebSocketState;
+      dispatchWebSocket: Dispatch<WebSocketAction>;
+    }
+  | undefined
+>(undefined);
+
+const WebSocketConfig = {
```

Comment:
Let's get these values from env: instead of certain ip and host, put getting env variable with corresponding name.
Before:
`address: "77.91.73.240",
  port: "8000",`
After:
`address: process.env.ADDRESS,
  port: process.env.PORT,`
Just replace the address and host as in the example above or you can create config file to stick all configurable files in one place